state-chain/pallets/cf-funding/src/lib.rs

```diff
@@ -343,7 +344,7 @@ pub mod pallet {
 
 			// Not allowed to redeem if we are an active bidder in the auction phase
 			if T::EpochInfo::is_auction_phase() {
-				T::BidderProvider::ensure_not_bidding(&account_id)?;
+				T::RedemptionChecker::ensure_can_redeem(&account_id)?;
```

Comment:
EpochInfo::is_auction_phase is defined in the validator pallet so we can move that check inside `ensure_can_redeem` too :)